python/py-igraph/Portfile

```diff
@@ -4,40 +4,39 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-igraph
```

Comment:
under the ```name``` field, you could add ```python.rootname     python-igraph```. If you do that, the python PG will populate some of the fields automatically and you can simplify the Portfile (i.e., remove some lines) - see comments below.